hooks/pre/30-upgrade.rb

```diff
@@ -74,12 +74,6 @@ def remove_nodes_distributors
   Kafo::Helpers.execute("mongo pulp_database --eval  'db.repo_distributors.remove({'distributor_type_id': \"nodes_http_distributor\"});'")
 end
 
-def fix_pulp_httpd_conf
-  return true unless File.exist?('/etc/httpd/conf.d/pulp.conf.rpmnew')
-
-  Kafo::Helpers.execute('cp /etc/httpd/conf.d/pulp.conf.rpmnew /etc/httpd/conf.d/pulp.conf')
```

Comment:
I honestly do not recall why it was added. Speculating that when the Pulp steps in the pre upgrade were being run, they would throw an error due to missing configuration. This would put this file back, allow that migration and such to happen and then let the installer properly handle this file.